packages/web/src/components/user-name-and-badges/UserNameAndBadges.tsx

```diff
@@ -45,3 +56,23 @@ export const UserNameAndBadges = (props: UserNameAndBadgesProps) => {
     </ArtistPopover>
   )
 }
+
```

Comment:
I had previously written a component for the Purchases/Sales table components that render user name and badges. But it turns out we already had one! I updated this to support the use case in sales/purchases (passing an id instead of a user record).
I will follow up in another PR to convert the tables over to using this and remove the redundant component